docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+services:
```

Comment:
We do usually order services as the should start-up so, better to declare MySQL service as first one that todolist with dependency on MySQL service